Vungle/VungleBannerCustomEvent.m

```diff
@@ -73,9 +76,7 @@ - (void)requestAdWithSize:(CGSize)size adapterInfo:(NSDictionary *)info adMarkup
 
 - (void)dealloc
 {
-    if (self.bannerState == BannerRouterDelegateStatePlaying) {
-        [[VungleSDK sharedSDK] finishDisplayingAd:self.placementId];
-    }
+    [[VungleRouter sharedRouter] completeBannerAdViewForDelegate:self];
```

Comment:
`completeBannerAdViewForDelegate` has the same functionality inside it.

I made this change because I think the custom events aren't supposed to communicate to the SDK directly, but only through the router. If this isn't the case, I can revert this back.

Reply:
> the custom events aren't supposed to communicate to the SDK directly, but only through the router

That's correct. I like your change.